datastore/admin-api/README.md

```diff
@@ -0,0 +1,38 @@
+# Google Cloud Datastore Admin Sample
+
+These samples demonstrate how to interact with the [Google Cloud Datastore Admin][Datastore] using C# and
+the .NET client libraries to call the Datastore Admin API.
+
+The samples requires [.NET Core 3.1][net-core] or later.  That means using
+[Visual Studio 2019](https://www.visualstudio.com/), or the command line.
+
+## Setup
+
+1.  Set up a [.NET development environment](https://cloud.google.com/dotnet/docs/setup).
+
+2.  Enable APIs for your project.
+    [Click here][enable-api] to visit Cloud Platform Console and enable the Google Stackdriver Cloud Datastore API.
+
+3. Set Environment Variables
```

Comment:
This step is need only to run our tests, but if the users copy paste the sample code in their projects, they don't need this. Just remove this step.